backbone/src/main/java/org/researchstack/backbone/utils/ResUtils.java

```diff
@@ -18,6 +18,21 @@
     public static final String ERROR_ICON                   = "rsb_error";
     public static final String IC_FINGERPRINT               = "rsb_fingerprint";
 
+    public static final String TREMOR_TEST_1                = "rsb_tremortest1";
```

Comment:
This suggestion totally goes against the rest of the code so take it for what it's worth... I like grouping constants with nested inner classes, So I can do something like ResUtils.Tremor.TEST_1.. or static import ResUtils.Tremor and have Studio auto-completing the constants relevant to Tremor.

public static class Tremor {
  public static final String TEST_1 = "rsb_tremortest1";
  public static final String TEST_2 = "rsb_tremortest2";
}